BT-Tracking/Services/AppSettings.swift

```diff
@@ -14,9 +14,12 @@ struct AppSettings {
     private enum Keys: String {
         case appState
         case appFirstTimeLaunched
+
+        case efgsEnabled
+
         case backgroundModeAlertShown
 
-        case lastProcessedFileName
+        case lastProcessedFileNames
```

Comment:
Nebude to chtit jeste nejakou migraci, aby se naztratil posledni CZ file name?

Reply:
Premyslel jsem nad tim, ale neni to tak dulezite, proste se stahnout jeste jednou za poslednich 14 dni, neprida to nic extra moc navic.